libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -22,7 +22,7 @@ export class FunctionClient implements IFunctionClient {
   public async invoke(payload?: string): Promise<string | undefined> {
     const command = new InvokeCommand({
       FunctionName: this.functionArn,
-      Payload: payload ? fromUtf8(JSON.stringify(payload)) : undefined,
+      Payload: fromUtf8(payload ? JSON.stringify(payload) : "null"),
```

Comment:
Add a message indicating that since we are stringifying the payload the string "null" cannot be injected. (I think)

Reply:
Added a comment. This is basically matching the sim's behavior where we treat json's `null` as `undefined`